lib/dictionaries/api/parameterizer.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function() {
+  return {
+    default: function(req, res, next) {
+      if(req.params.scope !== 'users' && req.params.scope !== 'accounts') {
+        res.status(400);
+        res.send({'error':'scope does not have a valid value'});
+        return;
```

Comment:
Maybe I would use if else here because that way you don't need to break the normal flow of the application with the return, which always worsens the readability.